policyengine_canada/variables/household/assets/household/household_assets.py

```diff
@@ -0,0 +1,10 @@
+from policyengine_canada.model_api import *
+
+
+class household_assets(Variable):
```

Comment:
```suggestion
class household_liquid_assets(Variable):
```
Or can you check how SPSD/M labels/organizes asset variables? Terms like `assets` and `property` are fairly general/vague.

Reply:
Please answer my question. Does personal property mean real estate only? We need to be precise.

Reply:
Hi Max, I didn't find any definition of property in the SPSDM.

So in the YAML file of Nova Scotia Income Assistance applicable assets, the list contains 2 values:
    - property_value
    - household_liquid_assets
    
For the [property_value], my documentation goes "equity in any real or personal property". I googled the definition of real property and personal property separately.

"Real property": means land whether within or outside Canada, including mines and minerals, and buildings, structures, improvements and other fixtures on, above or below the surface of the land, and includes an interest therein.
Definition from ["Federal Real Property Act and Federal Real Property Regulation"](https://www.canada.ca/en/treasury-board-secretariat/services/federal-real-property-management/guide-federal-real-property-act-regulation.html)

"Personal property": Tangible and intangible assets of a person other than its real property.
Definition from "[a Canada Practical Law Company](https://ca.practicallaw.thomsonreuters.com/2-575-1532?transitionType=Default&contextData=(sc.Default)&firstPage=true)"

So from what I understand, the house/real estate belongs to real property. Then I'd like to change the documentation into "equity in real property (land and attached assets) and personal property (tangible and intangible possessions apart from real estate)."

What do you think about this new version?

Reply:
@MaxGhenis Hi Max, could you please check this new description at your convenience? @

Reply:
Sorry for the delay. I think personal property is too broad. It's basically all assets (net worth ex debts).

Reply:
@MaxGhenis Hi Max. As defined in the source, the “**applicable assets**” include (1) **real property** and **personal property** and (2) **liquid assets.** So I believe the personal property here itself is good. I can change the variable name [**property_value**] to [**real_and_personal_property**], what do you think? Please kindly advice and thank you!